spec/requests/api/v1/driver_commission_histories_spec.rb

```diff
@@ -1,24 +1,64 @@
 require 'rails_helper'
 
 RSpec.describe '/api/v1/all', :type => :request do
```

Comment:
How do you feel about renaming this from 'all' to 'driver_commissions_history'? Im thinking that the describe would be describing the function name more precisely.